Snippets/Rabbit/Rabbit_4/Rabbit_4.csproj

```diff
@@ -28,11 +28,12 @@
     <Reference Include="NServiceBus.Core">
       <HintPath>..\..\..\packages\NServiceBus.6.0.0-rc0002\lib\net452\NServiceBus.Core.dll</HintPath>
     </Reference>
-    <Reference Include="NServiceBus.Transports.RabbitMQ">
-      <HintPath>..\..\..\packages\NServiceBus.RabbitMQ.4.0.0-beta0006\lib\net452\NServiceBus.Transports.RabbitMQ.dll</HintPath>
+    <Reference Include="NServiceBus.Transports.RabbitMQ, Version=4.0.0.0, Culture=neutral, PublicKeyToken=9fc386479f8a226c, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\NServiceBus.RabbitMQ.4.0.0-rc0001\lib\net452\NServiceBus.Transports.RabbitMQ.dll</HintPath>
+      <Private>True</Private>
     </Reference>
-    <Reference Include="RabbitMQ.Client, Version=3.6.5.0, Culture=neutral, PublicKeyToken=89e7d7c5feba84ce, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\RabbitMQ.Client.3.6.5\lib\net45\RabbitMQ.Client.dll</HintPath>
+    <Reference Include="RabbitMQ.Client, Version=4.0.0.0, Culture=neutral, PublicKeyToken=89e7d7c5feba84ce, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\RabbitMQ.Client.4.1.0-rc1\lib\net451\RabbitMQ.Client.dll</HintPath>
```

Comment:
@SimonCropp unfortunately our RC RabbitMQ transport depends on the RC of the RabbitMQ.Client. I.e. our RabbitMQ transport 4.0.0 cannnot go RTM until RabbitMQ.Client 4.1.0 goes RTM.

I think that our only option for now is to use the RabbitMQ.Client RC in our samples.